src/Commands/InstallCommand.php

```diff
@@ -37,19 +37,21 @@ protected function nativePhpPath()
 
     protected function installNpmDependencies()
     {
-        $this->executeCommand('npm set progress=false && npm install', $this->nativePhpPath());
-    }
-
-    protected function executeCommand($command, $path)
-    {
-        $process = (Process::fromShellCommandline($command, $path))->setTimeout(null);
-
-        if ('\\' !== DIRECTORY_SEPARATOR && file_exists('/dev/tty') && is_readable('/dev/tty')) {
-            $process->setTty(true);
-        }
-
-        $process->run(function ($type, $line) {
-            $this->output->write($line);
-        });
+        $phpBinPackageDir = config('nativephp.binary_package');
+        $nativeBinaryPath = $phpBinPackageDir . 'bin/' . (PHP_OS_FAMILY === 'Windows' ? 'win' : 'mac');
+        $this->info("NativePHP binary path: $nativeBinaryPath");
+        $this->info('Fetching latest dependencies…');
+        Process::path(__DIR__ . '/../../resources/js/')
+                ->env([
+                    'NATIVEPHP_PHP_BINARY_PATH' => base_path($nativeBinaryPath),
+                    'NATIVEPHP_CERTIFICATE_FILE_PATH' => base_path($phpBinPackageDir . 'cacert.pem'),
+                ])
+                ->forever()
+				->tty(PHP_OS_FAMILY != 'Windows')
```

Comment:
```suggestion
                ->tty(PHP_OS_FAMILY != 'Windows')
```

Reply:
Thanks for catching this. 
Open to feedback on ways to improve having VSCode adopt the coding style of the repo.

Still somewhat new contributing to open source.

Reply:
No worries, it's an easy mistake to make. Best thing you can do in VS Code is enable "Show invisible characters": https://tosbourn.com/how-to-turn-on-invisible-characters-in-vscode/#:~:text=In%20the%20search%20bar%20on,as%20turning%20off%20highlighting%20altogether.

Generally speaking though @mpociot is correct, the best solution is an automated one so that contributors and maintainers never have to think about (or discuss) the style guide.